### PR TITLE
Reset user-agent styles for selected element

### LINKED ISSUE #43: Copied text has yellow background when pasted into gmail or google docs
Can reproduce from here

http://codepen.io/nkbt/pen/eNPoQv?editors=0010

Weird stuff

copy-to-clipboard-yello-back

### PR DESCRIPTION
**Pull request by shvaikalesh**
Fixes #43.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -19,10 +19,11 @@ function copy(text, options) {
     range = document.createRange();
     selection = document.getSelection();
 
-    mark = document.createElement('mark');
+    mark = document.createElement('span');
     mark.textContent = text;
     mark.setAttribute('style', [
       // prevents scrolling to the end of the page
```
Comment (nkbt) [inline index.js]: i reckon this comment was meant for the next line

Comment (shvaikalesh) [inline index.js]: Ouch, thanks @nkbt

**Comment by nkbt** [comment]
Looks like selenium cannot connect to chrome. Usually it happens when chrome version is outdated, but I don't know more details here

**Comment by sudodoki** [comment]
E2E is still an issue. LGTM

**Pull request merged**